pkg/pillar/devicenetwork/dhcpcd.go

```diff
@@ -106,21 +106,24 @@ func doDhcpClientActivate(log *base.LogObject, nuc types.NetworkPortConfig) {
 		if nuc.Gateway != nil && nuc.Gateway.String() == "0.0.0.0" {
 			extras = append(extras, "--nogateway")
 		}
-		if !dhcpcdCmd(log, "--request", extras, nuc.IfName, true) {
-			log.Errorf("doDhcpClientActivate: request failed for %s\n",
-				nuc.IfName)
-		}
 		// Wait for a bit then give up
 		waitCount := 0
 		failed := false
-		for !dhcpcdExists(log, nuc.IfName) {
-			log.Warnf("dhcpcd %s not yet running", nuc.IfName)
+		for {
+			if !dhcpcdCmd(log, "--request", extras, nuc.IfName, true) {
+				log.Errorf("doDhcpClientActivate: request failed for %s\n",
+					nuc.IfName)
+			}
```

Comment:
What happens if the first one was just slow at starting the process? Will the second one fail and the first process keep running? Or can a subsequent --request cause a failure for the process which started slowly?
Should be able to verify by manually starting multiple dhcpcd --request for the same interface more or less concurrently and see what happens.